edge/pkg/edgehub/clients/quicclient/quicclient.go

```diff
@@ -59,7 +59,7 @@ func (qcc *QuicClient) Init() error {
 	tlsConfig := &tls.Config{
 		RootCAs:            pool,
 		Certificates:       []tls.Certificate{cert},
-		InsecureSkipVerify: true,
+		InsecureSkipVerify: false,
```

Comment:
the default value is false, wo we can remove this line

Reply:
if we do not set this, we need support two-way authentication between server and client

Reply:
yes, the mutual authentication already supported